src/main/java/com/sigopt/model/ConvergenceCriteria.java

```diff
@@ -0,0 +1,44 @@
+package com.sigopt.model;
+
+public class ConvergenceCriteria extends TrainingEarlyStoppingCriteria {
```

Comment:
I don't recommend this inheritance, it encodes serverside assumptions into the java client that may not be true later on

you could have a common base class between the two, but I'd actually just recommend they be separate classes for the most forward-extensibility

Reply:
also this class looks unused

Reply:
Switched over to inheriting directly form StructObject but before it was possible to pass this into the training monitor part of experiment create